src/factories/workflow-specific/PIM_WorkflowFactory_v1.sol

```diff
@@ -0,0 +1,289 @@
+
+        // if isRenouncedToken flag is set burn owner role, else transfer ownership to specified admin
+        if (PIMConfig.isRenouncedIssuanceToken) {
+            _transferTokenOwnership(issuanceToken, address(0));
+        } else {
+            _transferTokenOwnership(issuanceToken, PIMConfig.admin);
+        }
+
+        // if isRenouncedWorkflow flag is set factory keeps admin rights over workflow, else transfer admin rights to specified admin
+        if (PIMConfig.isRenouncedWorkflow) {
+            // record the admin as fee recipient eligible to claim buy/sell fees
+            _pimFeeRecipients[fundingManager] = PIMConfig.admin;
+        } else {
+            _transferWorkflowAdminRights(orchestrator, PIMConfig.admin);
+        }
+
+        emit IPIM_WorkflowFactory_v1.PIMWorkflowCreated(
+            fundingManager,
+            address(issuanceToken),
+            _msgSender(),
+            PIMConfig.recipient,
+            PIMConfig.isRenouncedIssuanceToken,
+            PIMConfig.isRenouncedWorkflow
+        );
+
+        return (orchestrator, issuanceToken);
+    }
+
+    //--------------------------------------------------------------------------
+    // Permissioned Functions
+
+    /// @inheritdoc IPIM_WorkflowFactory_v1
+    function withdrawPimFee(address fundingManager, address to)
+        external
+        onlyPimFeeRecipient(fundingManager)
+    {
+        uint amount =
+            IBondingCurveBase_v1(fundingManager).projectCollateralFeeCollected();
+        IBondingCurveBase_v1(fundingManager).withdrawProjectCollateralFee(
+            to, amount
+        );
+        emit IPIM_WorkflowFactory_v1.PimFeeClaimed(_msgSender(), amount);
+    }
+
+    /// @inheritdoc IPIM_WorkflowFactory_v1
+    function transferPimFeeEligibility(address fundingManager, address to)
+        external
+        onlyPimFeeRecipient(fundingManager)
+    {
+        _pimFeeRecipients[fundingManager] = to;
+        emit IPIM_WorkflowFactory_v1.PimFeeRecipientUpdated(_msgSender(), to);
+    }
+
+    //--------------------------------------------------------------------------
+    // Internal Functions
+
+    function _manageInitialSupplies(
+        IBondingCurveBase_v1 fundingManager,
+        IERC20 collateralToken,
+        ERC20Issuance_v1 issuanceToken,
+        uint initialCollateralSupply,
+        uint initialIssuanceSupply,
+        address recipient,
+        bool withInitialLiquidity
+    ) private {
+        if (withInitialLiquidity) {
+            // collateral token is paid for by the msg.sender
+            collateralToken.transferFrom(
+                _msgSender(), address(fundingManager), initialCollateralSupply
+            );
+            // issuance token is minted to the the specified recipient
+            issuanceToken.mint(recipient, initialIssuanceSupply);
+        } else {
+            // issuance token is minted to the burn address
+            issuanceToken.mint(address(0xDEAD), initialIssuanceSupply);
```

Comment:
More of a question as I just don't immediately understand this: Why is this minted to a non-accessible address? Is this just to get the curve to the right area of pricing, but the tokens shouldn't be used outside of that?